README.md

```diff
@@ -33,12 +33,20 @@ Building
 ```
 cmake -DCMAKE_BUILD_TYPE=Release . && make
 ```
-It requires cudart, curand >=8.0 and OpenMP 4.0 compatible compiler (**that is, not gcc <=4.8**).
+
+It requires cudart, curand 7.5, OpenMP 4.0 compatible compiler (**that is, not gcc <=4.8**) and 
```

Comment:
mismerge here: 7.5 vs >=8.0

Reply:
ah yeah that one went over my head, changing